### PR TITLE
update pd request/response by using message.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: server/tso_test.go [hunk 1]
```diff
@@ -38,14 +39,20 @@ func (s *testTsoSuite) TearDownSuite(c *C) {
 }
 
 func sendRequest(c *C, conn net.Conn, msgID uint64, request *pdpb.Request) {
```
Comment (siddontang) [inline server/tso_test.go]: I think we can use these help functions in other places to reduce the codes too.
Outer no need to care msgpb.Message.

Comment (qiuyesuifeng) [inline server/tso_test.go]: U mean create msg in WriteMessage function?

Comment (siddontang) [inline server/tso_test.go]: yes

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @disksing

**Comment by disksing** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**